tests/e2e/system_test/test_reclaim_space_cronjob.py

```diff
@@ -0,0 +1,212 @@
+        except Exception as e:
+            logger.exception(f"The PVC was not created, exception [{str(e)}]")
+            raise PVCNotCreated("PVC did not reach BOUND state.")
+
+        logger.info(f"PVC by name {pvc_obj.name} created")
+        self.pvc_objs_created.append(pvc_obj)
+
+        # wait since it make take some time for cronjobs to be deleted
+        time.sleep(timeout)
+        res = run_oc_command(cmd="get reclaimspacecronjob", namespace=namespace)
+        logger.info(f"Reclaim space jobs after CephFS PVC creation {res}")
+        assert (
+            len(res) == 1
+        ), f"After CephtFS PVC creation reclaim space cron job exists: {res}"
+        logger.info("No reclaim space cron job was created for CephFS PVCs")
+        pvc_obj.delete()
+
+    def test_skip_reclaim_space(self):
+        """
+        Test case to check that no reclaim space job is created for rbd pvc
+        in the openshift-* namespace if skipReclaimspaceSchedule value is True
+        """
+        timeout = 30
+        namespace = f"openshift-{uuid4().hex}"
+        self.namespace = namespace
+        with open(
+            os.path.join(
+                constants.TEMPLATE_CSI_ADDONS_DIR, "NamespaceSkipReclaim.yaml"
+            ),
+            "r",
+        ) as stream:
+            try:
+                namespace_yaml = yaml.safe_load(stream)
+                namespace_yaml["metadata"]["name"] = namespace
+            except yaml.YAMLError as exc:
+                logger.error(f"Can not read template yaml file {exc}")
+
+        temp_file = NamedTemporaryFile(
+            mode="w+", prefix="namespace_skip_reclaim_space", suffix=".yaml"
+        )
+        with open(temp_file.name, "w") as f:
+            yaml.dump(namespace_yaml, f)
+
+        self.temp_files_list.append(temp_file.name)
+
+        res = run_oc_command(cmd=f"create -f {temp_file.name}")
+        assert ERRMSG not in res[0], (
+            f"Failed to create namespace with name {namespace} " f"got result: {res}"
+        )
+        logger.info(f"Namespace {namespace} created")
+
+        try:
+            pvc_obj = helpers.create_pvc(
+                sc_name=constants.DEFAULT_STORAGECLASS_RBD,
+                size="1Gi",
+                namespace=namespace,
+            )
+        except Exception as e:
+            logger.exception(f"The PVC was not created, exception [{str(e)}]")
+            raise PVCNotCreated("PVC did not reach BOUND state.")
+
+        logger.info(f"PVC by name {pvc_obj.name} created")
+        self.pvc_objs_created.append(pvc_obj)
+
+        # wait since it may take some time for cronjobs to be created
+        time.sleep(timeout)
+        res = run_oc_command(cmd="get reclaimspacecronjob", namespace=namespace)
+        logger.info(f"Reclaim space jobs after RBD PVC creation {res}")
+        assert (
+            len(res) == 1
+        ), f"For RBD PVC creation reclaim space cron job exists: {res}"
```

Comment:
the list will be empty if the match is not found. 
we could also assert it as
assert not res, f"For RBD PVC creation reclaim space cron job exists: {res}"

Reply:
@PrasadDesala , the list returned by 'get reclaimspacecronjob' command will always contain column names ('name',  'scheduling' etc.). So list of lenght 1 means actually that no cron jobs found. But you are right, it is a little bit confusing, so I added a corresponding comment to the code.